tasks/91_make-a-person/make-a-person.js

```diff
@@ -0,0 +1,34 @@
+export class Person {
```

Comment:
It would be good to solve this problem using context of this class.
To define each attribute value and set it's value i constructor.